recompose-idea/src/main/kotlin/recompose/plugin/copypaste/ConvertXmlToComposeConfirmationDialog.kt

```diff
@@ -49,4 +54,11 @@ class ConvertXmlToComposeConfirmationDialog(project: Project) : DialogWrapper(pr
         setCancelButtonText(CommonBundle.getNoButtonText())
         return arrayOf(okAction, cancelAction)
     }
+
+    override fun doOKAction() {
+        if (donTShowThisCheckBox.isSelected) {
+            KotlinEditorOptions.getInstance().isDonTShowConversionDialog = true
```

Comment:
Is this reusing the flag of the Java -> Kotlin conversion? I assume we need our own flag and do not want to interfere with that?

Reply:
U suppose @t-regbs is conforming to what exists already in the KotlinEditorOptions: `isDonTShowConversionDialog` which does not make sense to me. I suppose `doNotShowThisCheckBoxAgain` or `dontShowThisCheckBoxAgain` will do. I would go for the former though.

Reply:
We cant use KotlinEditorOptions. That are the settings from the Kotlin Plugin. When we set "dontshow" to true, this will also hide the Java To Kotlin Dialog. And a setting on that dialog would also hide the recompose dialog. 
We need to create our own implementation of that class. https://github.com/JetBrains/kotlin/blob/master/idea/src/org/jetbrains/kotlin/idea/editor/KotlinEditorOptions.java

Reply:
Yes you're right @Foso 
I didn't realise that at the time